drafts/draft-0004.md

```diff
@@ -0,0 +1,160 @@
+
+### Example
+```toml
+[[CURRENCIES]]
+code="GOAT"
+issuer="GD5T6IPRNCKFOHQWT264YPKOZAWUMMZOLZBJ6BNQMUGPWGRLBK3U7ZNP"
+regulated=true
+validation_server="https://goat.io/tx_validate"
+approval_server="https://goat.io/tx_auth"
+approval_criteria= “The goat approval server will ensure that transactions are compliant with NFO regulation”
+```
+
+## Validation Server
+The transaction validation server receives unsigned transactions and checks for compliance. It can choose to accept the transaction, modify it to fit compliance, or reject it. 
+Request 
+HTTP POST request is encoded as application/json.
+
+Parameters :
+Name
+Data Type
+Description
+tx
+string
+Unsigned Transaction XDR, Base64 encoded.
+This transaction will be tested for compliance.
+Responses
+HTTP responses are encoded as application/json
+Success (status code 200)
+A successful response means that the transaction is compliant or the validation service was able to modify it to meet compliance.
+
+Parameters :
+Name
+Data Type
+Description
+tx
+string
+Unsigned Transaction XDR, Base64 encoded.
+This transaction is compliant.
+message
+string
+A human readable string containing information about the changes. made to the transaction in order to make it compliant (optional).
+Pending (status code 202)
+A pending response means that the issuer needs to asynchronously validate the transaction. The user should resubmit the transaction after a given timeout. 
+
+Parameters:
+Name
+Data Type
+Description
+timeout
+integer
+Number of milliseconds to wait before submitting the same transaction
+message
+string
+A human readable string containing information about the reason 
+Rejected (status code 400)
+A rejection response means that the transaction is not compliant and could not be made compliant.
+
+Parameters:
+Name
+Data Type
+Description
+error
+string
+A human readable string explaining why the transaction is not compliant and could not be made compliant.
+Best practices
+ALWAYS explain the changes that were made to a transaction through the message parameter.
+Core operations shouldn't be modified as that can be confusing and misleading. For example, if the user wishes to put an offer for 1000 GOATS but due to velocity limits they can only put an offer for 500 GOATS, it is better to error with a message than to change the amount.
+Adding an upper timebound to a transaction can help the issuer ensure that their view of the world does not get out of sync.
+Issuers can enforce additional fees by adding additional operations. For example, any transaction involving goats, will also send 0.1 GOAT to Boris’ account. 
+
+Approval Server
+The transaction approval server receives a transaction, verifies compliance and either signs or rejects it. 
+Request 
+HTTP POST request is encoded as application/json.
+
+Parameters :
+Name
+Data Type
+Description
+tx
+string
+Transaction XDR, Base64 encoded. Signed by account holder/s
```

Comment:
Currently, I don't care if the account holder sends me a signed TX for issuer signature, or an unsigned TX.  Either way, the TX can't be submitted without both.

Can this be an unsigned or signed transaction?